js/speakers.js

```diff
@@ -0,0 +1,49 @@
+// const seeMore = document.getElementById('see-more-speakers');
```

Comment:
- [ ] Please it would be better to remove the unused commented code in order to keep you file clean and pro